spec.html

```diff
@@ -32027,79 +32027,65 @@ <h1>Time Values and Time Range</h1>
-            = *2*<sub>𝔽</sub> if *59*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *90*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *3*<sub>𝔽</sub> if *90*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *120*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *4*<sub>𝔽</sub> if *120*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *151*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *5*<sub>𝔽</sub> if *151*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *181*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *6*<sub>𝔽</sub> if *181*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *212*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *7*<sub>𝔽</sub> if *212*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *243*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *8*<sub>𝔽</sub> if *243*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *273*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *9*<sub>𝔽</sub> if *273*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *304*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *10*<sub>𝔽</sub> if *304*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *334*<sub>𝔽</sub> + InLeapYear(_t_)
-            = *11*<sub>𝔽</sub> if *334*<sub>𝔽</sub> + InLeapYear(_t_) &le; DayWithinYear(_t_) &lt; *365*<sub>𝔽</sub> + InLeapYear(_t_)
+        <p>Months are identified by an integer in the range 0 to 11, inclusive. A month value of 0 specifies January; 1 specifies February; 2 specifies March; 3 specifies April; 4 specifies May; 5 specifies June; 6 specifies July; 7 specifies August; 8 specifies September; 9 specifies October; 10 specifies November; and 11 specifies December.</p>
+        <p>The position of a month _m_ within a year depends upon whether or not the year is a leap year:</p>
+        <emu-eqn id="eqn-DaysWithinYearBeforeMonth" aoid="DaysWithinYearBeforeMonth">DaysWithinYearBeforeMonth(_m_, _leapDaysInYear_)
+          = *+0*<sub>𝔽</sub> if _m_ = 0
+          = *31*<sub>𝔽</sub> if _m_ = 1
+          = *59*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 2
+          = *90*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 3
+          = *120*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 4
+          = *151*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 5
+          = *181*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 6
+          = *212*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 7
+          = *243*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 8
+          = *273*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 9
+          = *304*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 10
+          = *334*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 11
         </emu-eqn>
+        <emu-clause id="sec-MonthFromTime" aoid="MonthFromTime" oldids="eqn-MonthFromTime">
+          <h1>MonthFromTime ( _t_ )</h1>
+          <p>The abstract operation MonthFromTime maps a time value _t_ to a month number. It performs the following steps:</p>
```

Comment:
I'd recommend updating this definition to use a structured header with types so that you get the standard prose. (although I hope it's still possible to specify `oldids` on a structured header?)